website/src/pages/admin/messages/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { Card, CardBody } from "@chakra-ui/react";
+import { GetServerSideProps } from "next";
+import Head from "next/head";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { AdminArea } from "src/components/AdminArea";
+import { getAdminLayout } from "src/components/Layout";
+import { AdminMessageTable } from "src/components/Messages/AdminMessageTable";
+
+const MessageList = () => {
+  return (
+    <>
+      <Head>
+        <title>Open Assistant</title>
+      </Head>
+      <AdminArea>
+        <Card>
+          <CardBody>
+            <AdminMessageTable includeUser></AdminMessageTable>
+          </CardBody>
+        </Card>
+      </AdminArea>
+    </>
+  );
+};
+
+MessageList.getLayout = getAdminLayout;
+
+export default MessageList;
+
+export const getServerSideProps: GetServerSideProps = async ({ locale = "en" }) => {
+  return {
+    props: {
+      ...(await serverSideTranslations(locale, ["common", "labelling", "message"])),
+    },
+  };
+};
```

Comment:
would this work? https://github.com/LAION-AI/Open-Assistant/blob/3c4bf803b1e50d27cb767593433235ddb033662f/website/src/pages/messages/index.tsx#L8

Reply:
Done